website/src/urql.ts

```diff
@@ -6,14 +6,11 @@ import {
   fetchExchange,
   makeOperation,
 } from "urql"
+import { Environment, deploymentEnvironment } from "./env"
 
-export const apolloClient = (token: string | null) =>
+export const urqlClient = (token: string | null) =>
```

Comment:
Sorry I know this is annoying but since you're renaming this in this PR let's actually go for a more generic name because who knows if we'll switch again later. I'm thinking `graphqlClient` and the file name could be `website/src/graphql/client.ts` since we already have the `graphql` folder. Or if you have a better idea.

Reply:
Makes sense and I think this is a good idea! I went ahead and changed the names.